mock/mock_test.go

```diff
@@ -1323,6 +1323,32 @@ func (s *timer) GetTime(i int) string {
 	return s.Called(i).Get(0).(string)
 }
 
+type tCustomLogger struct {
+	*testing.T
+	err string
```

Comment:
how about we do
```
type tCustomLogger struct {
    *testing.T
    logs []string
}
```

That way we can use `logs` to have an entry in the array for each `Logf` call.